### DISCUSSION TITLE
YT Music Album art low resolution while Chromecasting

### DISCUSSION THREAD

**Discussion by patienttruth** [music-assistant/support#2053] [Q&A] [answered]
Hi, just got this going and was wondering if I can configure something to get full resolution cover art while Chromecasting from YT Music.

I get great looking art when playing music from my server, but it's pretty low quality when playing from YT Music. Any ideas?

![PXL_20240323_163536823](https://github.com/music-assistant/hass-music-assistant/assets/105557996/486edaaa-db3f-49b8-95e7-9ecce794d6b3)

**Comment by OzGav** [comment]
@MarvinSchenkel ?

**Reply by MarvinSchenkel** [reply, marked as answer]
Yeah this is expected. YTM is very inconsistent when it comes to delivering thumbnails. When I grab a Playlist or album from YTM, the thumbs for the tracks are usually in low quality for all songs. However, when you play a single track it should give you a HQ one. I try to work around the problem for albums and playlists by loading details for the next enqueued track, but some low quality album art is still expected.

**Reply by patienttruth** [reply]
Thank you both.
